### PR TITLE
Add a metric for DB writing throughput

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?
DB write IO is a key performance component.

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: node/metrics.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ type Metrics struct {
 	EigenMetrics eigenmetrics.Metrics
 	// Reachability gauge to monitoring the reachability of the node's retrieval/dispersal sockets
 	ReachabilityGauge *prometheus.GaugeVec
+	// The throughput (bytes per second) at which the data is written to database.
+	StoringThroughput prometheus.Gauge
```
Comment (dmanc) [inline node/metrics.go:49]: nit: Maybe DBWriteThroughout or StorageWriteThroughput?

File: node/metrics.go [hunk 2]
```diff
@@ -139,6 +141,14 @@ func NewMetrics(eigenMetrics eigenmetrics.Metrics, reg *prometheus.Registry, log
 			},
 			[]string{"service"},
 		),
+		StoringThroughput: promauto.With(reg).NewGauge(
+			prometheus.GaugeOpts{
+				Namespace: Namespace,
+				Name:      "storing_throughput",
```
Comment (dmanc) [inline node/metrics.go:147]: nit: Similar suggestions, maybe `storage_write_throughput_bytes_per_second` or `db_write_throughput_bytes_per_second`

**Review by dmanc — APPROVED** [review]

File: node/metrics.go (hunk 1 above)
Comment (jianoaix) [reply node/metrics.go]: Sg

File: node/metrics.go (hunk 2 above)
Comment (jianoaix) [reply node/metrics.go]: done

**Pull request merged**